packages/client/src/containers/sharedFiles/sharingOptions.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useEffect, useRef, useState } from 'react';
```

Comment:
In the react, using the uppercase as the first letter as react component name is a convention, our new and re-write component follow up this way 😉.

You can just execute `git mv packages/client/src/containers/sharedFiles/sharingOptions.tsx packages/client/src/containers/sharedFiles/SharingOptions.tsx` to change file name.